src/_P016_IR.ino

```diff
@@ -667,12 +679,21 @@ boolean Plugin_016(uint8_t function, struct EventStruct *event, String& string)
                 P016_data->AddCode(iCode, iCodeDecodeType, iCodeFlags); // add code if not saved so far
               }
 
+# ifdef P016_CHECK_HEAP
+  if (loglevelActiveFor(LOG_LEVEL_INFO)) {
+    addLogMove(LOG_LEVEL_INFO, strformat(
+      F("Before decode: FreeMem: %d  FreeStack:%d / After: FreeMem: %d  FreeStack:%d"),
+      fMem, fFreeStack,
+      FreeMem(), getCurrentFreeStack()));
+  }
+# endif // ifdef P016_CHECK_HEAP
+
```

Comment:
Hmm, not indenting the `#ifdef`'s, and not formatting the source using Uncrustify, horribly breaks the indentation guides (lines) in modern editors like VSCode... 🤔

Reply:
Sorry, I didn' uncrustify my changes. :sob:
On the other hand it is much easier for me to read the code if the `#ifdef's` start at the beginning of the line.

Reply:
> to read the code if the `#ifdef's` start at the beginning of the line.

Nope, not for me, that _totally_ breaks the normal flow of the code...

Reply:
I used the Uncrustify function in VScode, but position of the `#ifdef's` were not changed!

Reply:
That's not configured in the settings for Uncrustify 🥲 and I haven't found the courage to learn yet another complicated config...

Reply:
That means it has to be done manually?

Reply:
That's what I usually do, as I like my sourcecode prettified 😉

Reply:
I hope you enjoy my code now too :smiley: